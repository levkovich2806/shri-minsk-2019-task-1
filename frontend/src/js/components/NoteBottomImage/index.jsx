import React, { Component } from "react";
import styles from "./index.module.scss";

class NoteBottomImage extends Component {
  render() {
    const { url } = this.props;
    return (
      <div className={styles.content}>
        <a href={url} target="_blank" rel="noopener noreferrer">
          <img src={url} alt={url} />
        </a>
      </div>
    );
  }
}

export default NoteBottomImage;
