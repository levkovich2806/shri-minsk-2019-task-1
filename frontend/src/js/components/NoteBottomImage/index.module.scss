.content img {
  height: 64px;
  border: 1px solid rgba(218, 220, 224, 0.5);
  border-radius: 4px;
  margin-left: 24px;
}
