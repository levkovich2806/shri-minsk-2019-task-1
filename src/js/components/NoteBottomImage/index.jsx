import React, { Component } from "react";
import styles from "./index.module.scss";

class NoteBottomImage extends Component {
  render() {
    const { url } = this.props;
    return (
      <div className={styles.content}>
        <a href={url} target="_blank">
          <img src={url} />
        </a>
      </div>
    );
  }
}

export default NoteBottomImage;
