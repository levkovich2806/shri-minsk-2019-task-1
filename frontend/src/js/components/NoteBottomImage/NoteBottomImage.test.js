import React from "react";
import NoteBottomImage from "./index.jsx";
import renderer from "react-test-renderer";

describe("NoteBottomImage", function () {
  test("NoteBottomImage rendering", () => {

    const component = renderer.create(<NoteBottomImage url={'http://test.ru/logo.png'} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
