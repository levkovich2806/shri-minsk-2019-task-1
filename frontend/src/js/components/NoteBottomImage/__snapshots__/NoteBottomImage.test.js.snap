// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteBottomImage NoteBottomImage rendering 1`] = `
<div
  className="content"
>
  <a
    href="http://test.ru/logo.png"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      alt="http://test.ru/logo.png"
      src="http://test.ru/logo.png"
    />
  </a>
</div>
`;
